qa/suites/rados/singleton/all/max-pg-per-osd.yaml

```diff
@@ -0,0 +1,34 @@
+roles:
+- - mon.a
+  - mgr.x
+  - osd.0
+  - osd.1
+  - osd.2
+  - client.0
+openstack:
+  - volumes: # attached to each instance
+      count: 3
+      size: 10 # GB
+tasks:
+- install:
+- ceph:
+    config:
+      osd:
+        osd max pg per osd hard ratio : 1
+      global:
+        mon max pg per osd : 2
+    log-whitelist:
+      - \(TOO_FEW_PGS\)
+- exec:
+    client.0:
+      - sudo ceph osd pool create foo 2
+      - sudo ceph osd pool application enable foo rados
+      - sleep 5
+      - sudo ceph pg stat --format=json | jq '.num_pg_by_state | .[] | select(.name | contains("active+clean")) | .num == 2' | grep "true"
+      - sudo ceph osd pool create bar 2
+      - sudo ceph osd pool application enable bar rados
+      - sleep 5
+      - sudo ceph pg stat --format=json | jq '.num_pg_by_state | map(select(.name | contains("creating") or contains("unknown")) | .num) | add == 2' | grep "true"
+      - sudo ceph osd pool rm foo foo --yes-i-really-really-mean-it
+      - sleep 5
```

Comment:
these sleeps seem error prone, especially with msgr failure injection, and the possibility that the mon 'tell a random osd' will fail.  maybe make them longer, or loop, or ... is there an asok command to make the osd get the latest osdmap?

Reply:
will rewrite the test in py.